ignite/handlers/lr_finder.py

```diff
@@ -137,6 +137,17 @@ def _reset(self, trainer: Engine) -> None:
     def _log_lr_and_loss(self, trainer: Engine, output_transform: Callable, smooth_f: float, diverge_th: float) -> None:
         output = trainer.state.output
         loss = output_transform(output)
+        if not isinstance(loss, float):
+            if isinstance(loss, torch.Tensor) and (
+                (loss.ndimension() == 0) or (loss.ndimension() == 1 and len(loss) == 1)
+            ):
+                loss = loss.item()
+            else:
+                raise TypeError(
+                    "output of the engine should be of type float or 0d torch.Tensor "
+                    "or 1d torch.Tensor with 1 element, "
+                    f"but got output of type {type(loss).__name__}"
```

Comment:
This will show "Tensor" but I think we already know that loss is tensor, the point is to show shape maybe here.